tests/scripts/single_card/run_demos_single_card_wh_common.sh

```diff
@@ -12,6 +12,7 @@ WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest -n auto --disable-warn
 
 # working on both
 WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest -n auto models/demos/wormhole/mistral7b/demo/demo.py::test_demo[instruct_weights] --timeout 420
+WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest -n auto models/demos/wormhole/mistral7b/demo/demo_with_prefill.py::test_demo[general_weights] --timeout 420
```

Comment:
Could we take the opportunity to simplify this command and extract the demo out into a separate test that we can call like `WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest -n auto models/demos/wormhole/mistral7b/demo/demo_with_prefill.py` ?

Reply:
Yes. I'm working on that now. This will come with the use of the CI fixture 👍

Reply:
Just pushed changes addressing this for all Mistral tests and demo code. We will look into Mixtral in a separate PR.